packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -115,6 +115,14 @@ export const timeseriesChartSchema = {
     stacked: {
       type: 'boolean',
     },
+    threshold: {
+      type: 'object',
+      properties: Object.fromEntries(
+        exploreAggregations.map((key) => [key, { type: 'number' }]),
```

Comment:
This is what it yields:

<img width="328" alt="Screenshot 2024-12-04 at 08 44 04" src="https://github.com/user-attachments/assets/3267877b-9278-44c7-96e6-47efa039d0a4">

I assume it's preferable to dynamically build the allowed keys rather than spelling out each one, and having to maintain this list separate from `exploreAggregations`.

We can simply make it `{type: string}, {type: number}`, but that seems too imprecise.

Reply:
Dropped the `properties` and just ended up doing 

```
additionalProperties: {
      type: 'number',
 },
```